src/components/FormComponents/SourceWizardSummary.js

```diff
@@ -28,8 +28,8 @@ import {
 } from '../../components/addSourceWizard/schemaBuilder';
 import { useFlag } from '@unleash/proxy-client-react';
 
-const alertMapper = (appName, sourceType, intl) => {
-  if (appName === COST_MANAGEMENT_APP_NAME && sourceType !== 'google') {
+const alertMapper = (appName, sourceType, intl, hcsEnrolled) => {
```

Comment:
Can this be a regular React component? The argument list is starting to grow and its being used as a ReactNode